docs/airflow/production-maintenance.md

```diff
@@ -28,6 +28,7 @@ DAGs are listed in alphabetical order, as they appear in the Airflow UI.
 `airtable_loader` | **⛔ No*** | No | **🔂 No** | N/A | All tasks are unsafe after 24 hours |
 `airtable_views` | Yes | No | Yes* | `airtable_loader` | Latest-only data |
 `amplitude_benefits` | Yes | No | **🔂 No** | N/A | |
+`check_data_freshness` | Yes | No | N/A | N/A | |
```

Comment:
The 3rd column is generally only N/A for once-only DAGs... Can you add a note clarifying what that means in this context? Like is the implication just that it's not worth re-running because it runs often enough on its own?

Reply:
Oh yeah sure, and yes exactly; it's like the Composer uptime DAG